src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -265,55 +268,91 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
         }
     }
 
-    handleSaveSingleGiftEntry(sectionsList,enableSave,toggle) {
+    /*******************************************************************************
+    * @description Handles upserting of a Data Import record and firing a
+    * 'singlesubmit' event.
+    *
+    * @param {object} dataImportRecord: A DataImport__c record
+    * @param {object} formControls: An object holding methods that control
+    * the form save button enablement and lightning spinner toggler.
+    */
+    handleSaveSingleGiftEntry = async (dataImportRecord, formControls) => {
+        this.dataImport = await GeFormService.saveDataImport(dataImportRecord)
```

Comment:
Can saveDataImport be imported directly here?

Reply:
Yes, updated!